docs/databases.md

```diff
@@ -33,3 +39,34 @@ You will need to add a folder after `/db` with [may] contain the following ORM s
 - session stores
 - deserialising users
 
+##Setting up Postgres
+
+**Install Postgres as your database**:
+
+```bash
+# Update brew formulae
+brew update
+# Install Postgres
+brew install postgres
+```
+
+**Run your Postgres server**
+```bash
+postgres -D /usr/local/var/postgres
+```
+
+**Setup your postgres database**
+```bash
+createdb react_webpack_node_development
+sequelize db:migrate
```

Comment:
For local development, it might be worth noting that we need to run
```bash
createuser root
```
to create the role `root` that our default settings has.